packages/lib/server/queries/teams/index.ts

```diff
@@ -153,20 +154,20 @@ export async function getTeamWithMembers(args: {
     });
   }
 
-  const team = teams[0];
-  if (!team) return null;
+  const orgOrTeam = teams[0];
+  if (!orgOrTeam) return null;
 
   // This should improve performance saving already app data found.
   const appDataMap = new Map();
-  const members = team.members.map((obj) => {
+  const members = orgOrTeam.members.map((obj) => {
     const { credentials, ...restUser } = obj.user;
     return {
       ...restUser,
       role: obj.role,
       accepted: obj.accepted,
       disableImpersonation: obj.disableImpersonation,
       subteams: orgSlug
-        ? obj.user.teams.filter((obj) => obj.team.slug !== orgSlug).map((obj) => obj.team.slug)
+        ? obj.user.teams.filter((obj) => obj.team.id !== orgOrTeam.id).map((obj) => obj.team.slug)
```

Comment:
This is the fix. Instead of skipping the same slug team, we want to skip the org because every member would be a part of it.